core/src/main/java/org/springframework/security/authentication/AuthenticationTrustResolver.java

```diff
@@ -53,4 +53,17 @@ public interface AuthenticationTrustResolver {
 	 */
 	boolean isRememberMe(Authentication authentication);
 
+	/**
+	 * Indicates whether the passed <code>Authentication</code> token represents an anonymous user that
+	 * has been remembered (i.e. user that has been fully authenticated). This is a composition of 
+	 * <code>isAnonymous</code> and <code>isRememberMe</code> implementation
+	 * <p>
+	 * @param authentication to test (may be <code>null</code> in which case the method
+	 * will always return <code>false</code>)
+	 * @return <code>true</code> the passed authentication token represented an anonymous principal & is
+	 * authenticated using a remember-me token, <code>false</code> otherwise
```

Comment:
Will you please add `@since 5.8`? Remember to build with `./gradlew format` to ensure it is placed in the correct location in the JavaDoc.

Reply:
added `@since` annotation and formatted using `gradlew` task